src/sentry/snuba/functions.py

```diff
@@ -94,6 +94,7 @@ def timeseries_query(
         ),
     )
     results = builder.run_query(referrer)
+    results = builder.strip_alias_prefix(results)
```

Comment:
Is this an operation we could override `process_results` to have instead of requiring you to call this strip function explicitly?

Reply:
It's difficult to override `process_results` for this because it's not always used in some results. So modifying that would likely mean a lot more changes elsewhere which is why I made the changes to only the functions dataset